Dockerfile

```diff
@@ -1,8 +1,21 @@
+# Build go-neb
+FROM golang:1.8-alpine as builder
```

Comment:
why 1.8-alpine and not 1.10-alpine?

Reply:
Copy/pasta from the old script - will update